host_files/herbert/nginx/sonarqube.zeus.gent

```diff
@@ -0,0 +1,37 @@
+upstream sonarqube {
+   server 10.1.0.210:9000;
+   keepalive 32;
+}
+
+server {
+    listen 80;
+    server_name sonarqube.zeus.gent;
+
+    if ($request_method = GET) {
+        return 301 https://$server_name$request_uri;
+    }
+
+    return 308 https://$server_name$request_uri;
+}
+
+server {
+    listen 443 ssl http2;
+    server_name sonarqube.zeus.gent;
+
+    ssl_certificate     /root/.acme.sh/*.zeus.gent/fullchain.cer;
+    ssl_certificate_key /root/.acme.sh/*.zeus.gent/*.zeus.gent.key;
+    include snippets/ssl_options_preload.conf;
+
+   location / {
+       client_max_body_size 50M;
+       proxy_buffers 256 16k;
+       proxy_buffer_size 16k;
+       client_body_timeout 60;
+       send_timeout 300;
+       lingering_timeout 5;
+       proxy_connect_timeout 90;
+       proxy_send_timeout 300;
+       proxy_read_timeout 90s;
+       proxy_pass http://cat;
```

Comment:
```suggestion
       proxy_pass http://sonarqube;
```
Forgot to change this and now I'm doing it the right way

Reply:
Nice, I did a quick copy and paste and obviously missed most of the stuff I had to change :|